### PR TITLE
cam6_3_049:  Misc WACCM-X updates

### LINKED ISSUE #455: WACCM-X history fields EX, EY missing Rearth+z factor
The horizontal electric field history variables differs by ~1e7 between those on magnetic  coordinates (ED1, ED2) and geographic coordinates (EX, EY). See pdf. Discussion from Francis via email below. 

The difference in magnitudes seems to be a factor of Earth radius (meters).

        ed1(i,j) = -(phim2d(i+1,j)-phim2d(i-1,j))/(2._r8*dlonm*csth0)* &
          rcos0s(j)/(r0*1.e-2_r8)
        ephi(i,j) = ed1(i,j)*(r0*1.e-2_r8)

 - - r0 is earth radius in cm

History field ED1 is the output of ed1 on the magnetic grid and history field EX is the output of ephi on the geographic grid.  I don't think the units of EX are correct.  

The electric field is divided by distance from the center of the Earth when ion velocities are calculated (3D ExB):

          eex(k,i) = (rjac(ii,j,1,1)*ex(k,i,j)+ &
                      rjac(ii,j,2,1)*ey(k,i,j))/(Rearth+z(k,i,j))

        Rearth = rearth_m*100._r8,  & ! earth radius (cm)
        h0 = 9.7e6_r8,              & ! minimum height (cm)
        r0 = Rearth + h0,           & ! min height from earth center

It might be more useful to output eex and eey which are the east and west components of the electric field in geographic coordinates.
[electric field variables in WACCM-X.pdf](https://github.com/ESCOMP/CAM/files/7427729/electric.field.variables.in.WACCM-X.pdf)

So not only are EX, EY off by a height dependent factor, they are not actually pointing in the right direction.

### LINKED ISSUE #456: WACCM-X conductance field seems to have a "hole" over geographic pole
WACCM-X diagnostic field EDYN_ZIGM11_PED appears to have "holes" over the geographic poles.

The adot terms that go it to zigm11 are zero at the poles. The calculation of the adot terms in adotv_mod.F90 skips the pole points.   This is inconsistent with TIEGCM.

### LINKED ISSUE #457: Need to reduce number of WACCMX history fields in the master field list
There are a number of history fields that were added to aid in the development of WACCMX.  Many of these fields showing up in the CAM's master field list have cause confusion to external users.  A number of the following fields should be removed from code or added only within a debugging or development conditional.  

```
 addfld ('DPIE_OMEGA',(/ 'lev' /), 'I', 'Pa/s    ','OMEGA input to DPIE coupling', gridname='physgrid')
 addfld ('DPIE_MBAR' ,(/ 'lev' /), 'I', 'kg/kmole','MBAR Mean Mass from dpie_coupling', gridname='physgrid')
 addfld ('DPIE_TN   ',(/ 'lev' /), 'I', 'deg K   ','DPIE_TN'   , gridname='physgrid')
 addfld ('DPIE_UN   ',(/ 'lev' /), 'I', 'cm/s    ','DPIE_UN'   , gridname='physgrid')
 addfld ('DPIE_VN   ',(/ 'lev' /), 'I', 'cm/s    ','DPIE_VN'   , gridname='physgrid')
 addfld ('DPIE_WN   ',(/ 'lev' /), 'I', 'cm/s    ','DPIE_WN'   , gridname='physgrid')
 addfld ('DPIE_OM   ',(/ 'lev' /), 'I', 's-1     ','DPIE_OM'   , gridname='physgrid')
 addfld ('DPIE_ZHT  ',(/ 'lev' /), 'I', 'cm      ','DPIE_ZHT (geometric height,simple)', gridname='physgrid')
 addfld ('DPIE_ZGI  ',(/ 'lev' /), 'I', 'cm      ','DPIE_ZGI (geopotential height on interfaces)', gridname='physgrid')
 addfld ('DPIE_O2   ',(/ 'lev' /), 'I', 'mmr     ','DPIE_O2'   , gridname='physgrid')
 addfld ('DPIE_O    ',(/ 'lev' /), 'I', 'mmr     ','DPIE_O'    , gridname='physgrid')
 addfld ('DPIE_N2   ',(/ 'lev' /), 'I', 'mmr     ','DPIE_N2'   , gridname='physgrid')
 addfld ('DPIE_TE   ',(/ 'lev' /), 'I', 'deg K   ','DPIE_TE'   , gridname='physgrid')
 addfld ('DPIE_TI   ',(/ 'lev' /), 'I', 'deg K   ','DPIE_TI'   , gridname='physgrid')
 addfld ('DPIE_OPMMR' ,(/ 'lev' /), 'I', 'mmr'  ,'DPIE_OPMMR'  , gridname='physgrid')
 addfld ('DPIE_O2P',(/ 'lev' /), 'I', 'm^-3','DPIE_O2P(dpie input)', gridname='physgrid')
 addfld ('DPIE_NOP',(/ 'lev' /), 'I', 'm^-3','DPIE_NOP(dpie input)', gridname='physgrid')
 addfld ('DPIE_N2P',(/ 'lev' /), 'I', 'm^-3','DPIE_N2P(dpie input)', gridname='physgrid')
 addfld ('HMF2'       , horiz_only , 'I', 'km'  ,'Height of the F2 Layer'      , gridname='physgrid')
 addfld ('NMF2'       , horiz_only , 'I', 'cm-3','Peak Density of the F2 Layer', gridname='physgrid')
 addfld ('OpDens' ,(/ 'lev' /), 'I', 'cm^-3','O+ Number Density'                       , gridname='physgrid')
 addfld ('EDens'  ,(/ 'lev' /), 'I', 'cm^-3','e Number Density (sum of O2+,NO+,N2+,O+)', gridname='physgrid')
 addfld ('prescr_efxp'  , horiz_only, 'I','mW/m2','Prescribed energy flux on geo grid'     ,gridname='physgrid')
 addfld ('prescr_kevp'  , horiz_only, 'I','keV  ','Prescribed mean energy on geo grid'     ,gridname='physgrid')
 addfld ('WACCM_UI'   ,(/ 'lev' /), 'I', 'm/s'  ,'WACCM_UI (dpie output)', gridname='physgrid')
 addfld ('WACCM_VI'   ,(/ 'lev' /), 'I', 'm/s'  ,'WACCM_VI (dpie output)', gridname='physgrid')
 addfld ('WACCM_WI'   ,(/ 'lev' /), 'I', 'm/s'  ,'WACCM_WI (dpie output)', gridname='physgrid')
 addfld ('WACCM_OP'   ,(/ 'lev' /), 'I', 'kg/kg'  ,'WACCM_OP (dpie output)', gridname='physgrid')

 addfld ('PED_MAG'   ,(/ 'lev' /), 'I', 'S/m  ','Pedersen Conductivity'            ,gridname='gmag_grid')
 addfld ('HAL_MAG'   ,(/ 'lev' /), 'I', 'S/m  ','Hall Conductivity'                ,gridname='gmag_grid')
 addfld ('PHIM2D'    , horiz_only, 'I', 'VOLTS','PHIM2D: Electric Potential'       ,gridname='gmag_grid')
 addfld ('ED1'       , horiz_only, 'I', 'V/m  ','ED1: Eastward Electric Field'     ,gridname='gmag_grid')
 addfld ('ED2'       , horiz_only, 'I', 'V/m  ','ED2: Equatorward Electric Field'  ,gridname='gmag_grid')
 addfld ('PHIM3D'    ,(/ 'lev' /), 'I', 'VOLTS','PHIM3D: 3d Electric Potential'    ,gridname='gmag_grid')
 addfld ('EPHI3D'    ,(/ 'lev' /), 'I', ' ','EPHI3D'    ,gridname='gmag_grid')
 addfld ('ELAM3D'    ,(/ 'lev' /), 'I', ' ','ELAM3D'    ,gridname='gmag_grid')
 addfld ('EMZ3D'     ,(/ 'lev' /), 'I', ' ','EMZ3D'     ,gridname='gmag_grid')
 addfld ('ED13D'     ,(/ 'lev' /), 'I', 'V/m  ','ED13D: Eastward Electric Field'   ,gridname='gmag_grid')
 addfld ('ED23D'     ,(/ 'lev' /), 'I', 'V/m  ','ED23D: Equatorward Electric Field',gridname='gmag_grid')
 addfld ('ZPOT_MAG'  ,(/ 'lev' /), 'I', 'cm   ','Geopotential on mag grid (h0 min)',gridname='gmag_grid')
 addfld ('ADOTV1_MAG',(/ 'lev' /), 'I', '     ','ADOTV1 on mag grid'               ,gridname='gmag_grid')
 addfld ('ADOTV2_MAG',(/ 'lev' /), 'I', '     ','ADOTV2 on mag grid'               ,gridname='gmag_grid')
 addfld ('prescr_phihm' , horiz_only, 'I','VOLTS','Prescribed Electric Potential-mag grid' ,gridname='gmag_grid')
 addfld ('prescr_efxm'  , horiz_only, 'I','mW/m2','Prescribed energy flux on mag grid'     ,gridname='gmag_grid')
 addfld ('prescr_kevm'  , horiz_only, 'I','keV  ','Prescribed mean energy on mag grid'     ,gridname='gmag_grid')
 addfld ('EDYN_ADOTV1 ',(/ 'lev' /), 'I', '        ','EDYN_ADOTV1' , gridname='geo_grid')
 addfld ('EDYN_ADOTV2 ',(/ 'lev' /), 'I', '        ','EDYN_ADOTV2' , gridname='geo_grid')
 addfld ('EDYN_SINI   ', horiz_only , 'I', '        ','EDYN_SINI'   , gridname='geo_grid')
 addfld ('EDYN_ADOTA1 ', horiz_only , 'I', '        ','EDYN_ADOTA1' , gridname='geo_grid')
 addfld ('EDYN_ADOTA2 ', horiz_only , 'I', '        ','EDYN_ADOTA2' , gridname='geo_grid')
 addfld ('EDYN_A1DTA2 ', horiz_only , 'I', '        ','EDYN_A1DTA2' , gridname='geo_grid')
 addfld ('EDYN_BE3    ', horiz_only , 'I', '        ','EDYN_BE3'    , gridname='geo_grid')
 addfld ('ADOTA1_MAG', horiz_only , 'I', ' ','ADOTA1 in geo-mag coords' , gridname='gmag_grid')
 addfld ('SINI_MAG',   horiz_only , 'I', ' ','sini in geo-mag coords' , gridname='gmag_grid')
 addfld ('adota1_mag_a', horiz_only, 'I', ' ','EDYN_ZIGM11',gridname='gmag_grid')
 addfld ('ZIGM11_a', horiz_only, 'I', ' ','EDYN_ZIGM11',gridname='gmag_grid')
 addfld ('EDYN_ZIGM11_0', horiz_only, 'I', ' ','EDYN_ZIGM11',gridname='gmag_grid')
 addfld ('EDYN_ZIGM11', horiz_only, 'I', ' ','EDYN_ZIGM11',gridname='gmag_grid')
 addfld ('EDYN_ZIGM11_PED', horiz_only, 'I', 'S','Pedersen Conductance',gridname='gmag_grid')
 addfld ('EDYN_ZIGM22', horiz_only, 'I', ' ','EDYN_ZIGM22',gridname='gmag_grid')
 addfld ('EDYN_ZIGMC' , horiz_only, 'I', ' ','EDYN_ZIGMC' ,gridname='gmag_grid')
 addfld ('EDYN_ZIGM2' , horiz_only, 'I', ' ','EDYN_ZIGM2' ,gridname='gmag_grid')
 addfld ('EDYN_ZIGM2_HAL' , horiz_only, 'I', 'S','Hall Conductance' ,gridname='gmag_grid')
 addfld ('EDYN_RIM1'  , horiz_only, 'I', ' ','EDYN_RIM1'  ,gridname='gmag_grid')
 addfld ('EDYN_RIM2'  , horiz_only, 'I', ' ','EDYN_RIM2'  ,gridname='gmag_grid')
 addfld ('POTEN'  ,(/ 'lev' /), 'I', 'Volts','POTEN: Electric Potential',&
 addfld ('EX'     ,(/ 'lev' /), 'I', 'V/m'  ,'EX: Geographic zonal component of Electric Field',&
 addfld ('EY'     ,(/ 'lev' /), 'I', 'V/m'  ,'EY: Geographic meridional component of Electric Field',&
 addfld ('EZ'     ,(/ 'lev' /), 'I', 'V/m'  ,'EZ: Vertical component of Electric Field',&
 addfld ('BMOD', horiz_only, 'I', 'gauss','magnitude of magnetic field',gridname='geo_grid')
 addfld ('XB',   horiz_only, 'I', 'gauss','northward component of magnetic field',gridname='geo_grid')
 addfld ('YB',   horiz_only, 'I', 'gauss','eastward component of magnetic field',gridname='geo_grid')
 addfld ('ZB',   horiz_only, 'I', 'gauss','downward component of magnetic field',gridname='geo_grid')
 addfld ('RJAC11',(/'lev'/), 'I', '1','cos(thetas)/cos(theta)*d(lamdas)/d(lamda)'  ,gridname='geo_grid')
 addfld ('RJAC12',(/'lev'/), 'I', '1','cos(thetas)*d(lamdas)/d(theta)'  ,gridname='geo_grid')
 addfld ('RJAC21',(/'lev'/), 'I', '1','1./cos(theta)*d(thetas)/d(lamda)'  ,gridname='geo_grid')
 addfld ('RJAC22',(/'lev'/), 'I', '1','d(thetas)/d(theta)'  ,gridname='geo_grid')
 addfld ('OPLUS', (/ 'lev' /), 'I', 'cm^3','O+ (oplus_xport output)',    gridname='geo_grid')
 addfld ('OPtm1i',(/ 'lev' /), 'I', 'cm^3','O+ (oplus_xport output)',    gridname='geo_grid')
 addfld ('OPtm1o',(/ 'lev' /), 'I', 'cm^3','O+ (oplus_xport output)',    gridname='geo_grid')
 addfld ('PED_phys',(/ 'lev' /), 'I', 'S/m','Pedersen Conductivity'  , gridname='physgrid')
 addfld ('HAL_phys',(/ 'lev' /), 'I', 'S/m','Hall Conductivity'   , gridname='physgrid')

 addfld('OpTM1&IC', (/ 'lev' /), 'I', 'kg/kg',                   &
 addfld ('UI',(/ 'lev' /),'I','m/s', 'UI Zonal ion drift from edynamo')
 addfld ('VI',(/ 'lev' /),'I','m/s', 'VI Meridional ion drift from edynamo')
 addfld ('WI',(/ 'lev' /),'I','m/s', 'WI Vertical ion drift from edynamo')
 addfld ('UI&IC', (/ 'lev' /), 'I','m/s', 'Zonal ion drift velocity')
 addfld ('VI&IC', (/ 'lev' /), 'I','m/s', 'Meridional ion drift velocity')
 addfld ('WI&IC', (/ 'lev' /), 'I','m/s', 'Vertical ion drift velocity')
 addfld ('amie_efx_phys', horiz_only, 'I', 'mW/m2', 'AMIE energy flux')
 addfld ('amie_kev_phys', horiz_only, 'I', 'keV',   'AMIE mean energy')
 addfld ('ltr_efx_phys', horiz_only, 'I', 'mW/m2', 'LTR energy flux')
 addfld ('ltr_kev_phys', horiz_only, 'I', 'keV',  'LTR mean energy')

 addfld ('Z3GM',       (/ 'lev' /), 'I', 'm',                       &
 addfld ('Z3GMI',      (/ 'lev' /), 'I', 'm',                       &

 addfld ('OPLUS_Z'    ,(/ 'lev' /), 'I', 'cm   ','OPLUS_Z'     , gridname='geo_grid')
 addfld ('OPLUS_TN'   ,(/ 'lev' /), 'I', 'deg K','OPLUS_TN'    , gridname='geo_grid')
 addfld ('OPLUS_TE'   ,(/ 'lev' /), 'I', 'deg K','OPLUS_TE'    , gridname='geo_grid')
 addfld ('OPLUS_TI'   ,(/ 'lev' /), 'I', 'deg K','OPLUS_TI'    , gridname='geo_grid')
 addfld ('OPLUS_UN'   ,(/ 'lev' /), 'I', 'cm/s' ,'OPLUS_UN'    , gridname='geo_grid')
 addfld ('OPLUS_VN'   ,(/ 'lev' /), 'I', 'cm/s' ,'OPLUS_VN'    , gridname='geo_grid')
 addfld ('OPLUS_OM'   ,(/ 'lev' /), 'I', 'Pa/s' ,'OPLUS_OM'    , gridname='geo_grid')
 addfld ('OPLUS_O2'   ,(/ 'lev' /), 'I', 'mmr'  ,'OPLUS_O2'    , gridname='geo_grid')
 addfld ('OPLUS_O1'   ,(/ 'lev' /), 'I', 'mmr'  ,'OPLUS_O1'    , gridname='geo_grid')
 addfld ('OPLUS_N2'   ,(/ 'lev' /), 'I', 'mmr'  ,'OPLUS_N2'    , gridname='geo_grid')
 addfld ('OPLUS_OP'   ,(/ 'lev' /), 'I', 'cm^3' ,'OPLUS_OP'    , gridname='geo_grid')
 addfld ('OPLUS_UI'   ,(/ 'lev' /), 'I', 'm/s'  ,'OPLUS_UI'    , gridname='geo_grid')
 addfld ('OPLUS_VI'   ,(/ 'lev' /), 'I', 'm/s'  ,'OPLUS_VI'    , gridname='geo_grid')
 addfld ('OPLUS_WI'   ,(/ 'lev' /), 'I', 'm/s'  ,'OPLUS_WI'    , gridname='geo_grid')
 addfld ('OPLUS_MBAR' ,(/ 'lev' /), 'I', ' '    ,'OPLUS_MBAR'  , gridname='geo_grid')
 addfld ('OPLUS_TR'   ,(/ 'lev' /), 'I', ' '    ,'OPLUS_TR'    , gridname='geo_grid')
 addfld ('OPLUS_TP0'  ,(/ 'lev' /), 'I', ' '    ,'OPLUS_TP0'   , gridname='geo_grid')
 addfld ('OPLUS_TP1'  ,(/ 'lev' /), 'I', ' '    ,'OPLUS_TP1'   , gridname='geo_grid')
 addfld ('OPLUS_DJ'   ,(/ 'lev' /), 'I', ' '    ,'OPLUS_DJ'    , gridname='geo_grid')
 addfld ('OPLUS_HJ'   ,(/ 'lev' /), 'I', ' '    ,'OPLUS_HJ'    , gridname='geo_grid')
 addfld ('OPLUS_BVEL' ,(/ 'lev' /), 'I', ' '    ,'OPLUS_BVEL'  , gridname='geo_grid')
 addfld ('OPLUS_DIFFJ',(/ 'lev' /), 'I', ' '    ,'OPLUS_DIFFJ' , gridname='geo_grid')
 addfld ('OPLUS_OPNM' ,(/ 'lev' /), 'I', ' '    ,'OPLUS_OPNM'  , gridname='geo_grid')
 addfld ('OPNM_SMOOTH',(/ 'lev' /), 'I', ' '    ,'OPNM_SMOOTH' , gridname='geo_grid')
 addfld ('BDOTDH_OP'  ,(/ 'lev' /), 'I', ' '    ,'BDOTDH_OP'   , gridname='geo_grid')
 addfld ('BDOTDH_OPJ' ,(/ 'lev' /), 'I', ' '    ,'BDOTDH_OPJ'  , gridname='geo_grid')
 addfld ('BDOTDH_DIFF',(/ 'lev' /), 'I', ' '    ,'BDOTDH_DIFF' , gridname='geo_grid')
 addfld ('BDZDVB_OP'  ,(/ 'lev' /), 'I', ' '    ,'BDZDVB_OP'   , gridname='geo_grid')
 addfld ('EXPLICIT0'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICIT0'   , gridname='geo_grid')
 addfld ('EXPLICITa'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICITa'   , gridname='geo_grid') ! part a
 addfld ('EXPLICITb'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICITb'   , gridname='geo_grid') ! part b
 addfld ('EXPLICIT1'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICIT1'   , gridname='geo_grid') ! complete
 addfld ('EXPLICIT'   ,(/ 'lev' /), 'I', ' '    ,'EXPLICIT'    , gridname='geo_grid') ! final w/ poles
 addfld ('EXPLICIT2'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICIT2'   , gridname='geo_grid')
 addfld ('EXPLICIT3'  ,(/ 'lev' /), 'I', ' '    ,'EXPLICIT3'   , gridname='geo_grid')
 addfld ('TPHDZ0'     ,(/ 'lev' /), 'I', ' '    ,'TPHDZ0'      , gridname='geo_grid')
 addfld ('TPHDZ1'     ,(/ 'lev' /), 'I', ' '    ,'TPHDZ1'      , gridname='geo_grid')
 addfld ('DIVBZ'      ,(/ 'lev' /), 'I', ' '    ,'DIVBZ'       , gridname='geo_grid')
 addfld ('HDZMBZ'     ,(/ 'lev' /), 'I', ' '    ,'HDZMBZ'      , gridname='geo_grid')
 addfld ('HDZPBZ'     ,(/ 'lev' /), 'I', ' '    ,'HDZPBZ'      , gridname='geo_grid')
 addfld ('P_COEFF0'   ,(/ 'lev' /), 'I', ' '    ,'P_COEFF0'    , gridname='geo_grid')
 addfld ('Q_COEFF0'   ,(/ 'lev' /), 'I', ' '    ,'Q_COEFF0'    , gridname='geo_grid')
 addfld ('R_COEFF0'   ,(/ 'lev' /), 'I', ' '    ,'R_COEFF0'    , gridname='geo_grid')
 addfld ('P_COEFF0a'  ,(/ 'lev' /), 'I', ' '    ,'P_COEFF0a'   , gridname='geo_grid')
 addfld ('Q_COEFF0a'  ,(/ 'lev' /), 'I', ' '    ,'Q_COEFF0a'   , gridname='geo_grid')
 addfld ('DJINT'      ,(/ 'lev' /), 'I', ' '    ,'DJINT'       , gridname='geo_grid')
 addfld ('BDOTU'      ,(/ 'lev' /), 'I', ' '    ,'BDOTU'       , gridname='geo_grid')
 addfld ('R_COEFF0a'  ,(/ 'lev' /), 'I', ' '    ,'R_COEFF0a'   , gridname='geo_grid')
 addfld ('P_COEFF1'   ,(/ 'lev' /), 'I', ' '    ,'P_COEFF1'    , gridname='geo_grid')
 addfld ('Q_COEFF1'   ,(/ 'lev' /), 'I', ' '    ,'Q_COEFF1'    , gridname='geo_grid')
 addfld ('R_COEFF1'   ,(/ 'lev' /), 'I', ' '    ,'R_COEFF1'    , gridname='geo_grid')
 addfld ('P_COEFF2'   ,(/ 'lev' /), 'I', ' '    ,'P_COEFF2'    , gridname='geo_grid')
 addfld ('Q_COEFF2'   ,(/ 'lev' /), 'I', ' '    ,'Q_COEFF2'    , gridname='geo_grid')
 addfld ('R_COEFF2'   ,(/ 'lev' /), 'I', ' '    ,'R_COEFF2'    , gridname='geo_grid')
 addfld ('P_COEFF'    ,(/ 'lev' /), 'I', ' '    ,'P_COEFF'     , gridname='geo_grid') ! final w/ poles
 addfld ('Q_COEFF'    ,(/ 'lev' /), 'I', ' '    ,'Q_COEFF'     , gridname='geo_grid') ! final w/ poles
 addfld ('R_COEFF'    ,(/ 'lev' /), 'I', ' '    ,'R_COEFF'     , gridname='geo_grid') ! final w/ poles
 addfld ('OP_SOLVE'   ,(/ 'lev' /), 'I', ' '    ,'OP_SOLVE'    , gridname='geo_grid')
 addfld ('op_dt'  ,    (/ 'lev' /), 'I', ' '    ,'op_dt'       , gridname='geo_grid')
 addfld ('amb_diff'  , (/ 'lev' /), 'I', ' '    ,'amb_diff'    , gridname='geo_grid')
 addfld ('dfield'  ,   (/ 'lev' /), 'I', ' '    ,'dfield'      , gridname='geo_grid')
 addfld ('dwind'  ,    (/ 'lev' /), 'I', ' '    ,'dwind'       , gridname='geo_grid')
 addfld ('OP_OUT'     ,(/ 'lev' /), 'I', 'cm^3' ,'OPLUS (oplus_xport output)', gridname='geo_grid')
 addfld ('OPNM_OUT'   ,(/ 'lev' /), 'I', 'cm^3' ,'OPNM_OUT'    , gridname='geo_grid')
 addfld ('BMOD2'      ,(/ 'lev' /), 'I', ' '    ,'BMOD2'       , gridname='geo_grid')
 addfld ('OPLUS_FLUX', horiz_only , 'I', ' ','OPLUS_FLUX', gridname='geo_grid')
 addfld ('OPLUS_DIVB', horiz_only , 'I', ' ','OPLUS_DIVB', gridname='geo_grid')
 addfld ('OPLUS_BX'  , horiz_only , 'I', ' ','OPLUS_BX'  , gridname='geo_grid')
 addfld ('OPLUS_BY'  , horiz_only , 'I', ' ','OPLUS_BY'  , gridname='geo_grid')
 addfld ('OPLUS_BZ'  , horiz_only , 'I', ' ','OPLUS_BZ'  , gridname='geo_grid')
 addfld ('OPLUS_BMAG', horiz_only , 'I', ' ','OPLUS_BMAG', gridname='geo_grid')
```

@hanli-liu, @joemci, @npedatella, @dan800 
Please help narrow down the list.

### LINKED ISSUE #465: D-Region ion chemistry in WACCMX not converging
The chemistry solver is not able to converge at high altitudes in WACCMX when D-region ion mechanism is used.  The model is very slow and produces copious "failed to converge" error messages, such as:

75: imp_sol: Time step   1.8750000000000E+00 failed to converge @ (lchnk,lev,col,nstep) =    645    26    12   226
75: imp_sol: Failed to converge @ (lchnk,lev,col,nstep,dt,time) =    645    26    12   226  1.8750000000000E+00  1.3687500000000E+02
75: CO3m      0.000E+00
75: CO4m      0.000E+00
75: NOp_CO2   0.000E+00

CO2 is explicitly solved while the ions are implicitly solved.

CO2 and some positive ions are quickly driven to zero in some regions which seems to cause the implicit solver to not converge on a solultion:

co2_exp_00
NOp_CO2_exp_00

When implicitly solving CO2, along with the ions, the "failed to converge" message do not appear and the model runs normally.

Following are corresponding solutions when implicitly solving for CO2: 

co2_imp_00
NOp_CO2_imp_00

The WACCMX compsets we currently have with D-region ion chemistry are:

   FXmadHIST            : HIST_CAM60%WXIED_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV
   FXmadSD              : HIST_CAM60%WXIED%SDYN_CLM50%SP_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV

These use the waccm_mad_mam4 chemistry mechanism.  It seems that we should move CO2 to the implicit solver in this mechanism if want to support these WACCMX compsets.

@dan800 , Thoughts?

### PR DESCRIPTION
**Pull request by fvitt**
This PR is a collection of WACCM-X bug fixes / updates and improved default PE layouts on Cheyenne.

Addresses github issues:
 . D-Region ion chemistry in WACCMX not converging #465
 . WACCM-X conductance field seems to have a "hole" over geographic pole #456
 . WACCM-X history fields EX, EY missing Rearth+z factor #455
 . Need to reduce number of WACCMX history fields in the master field list #457

Fixes #455 
Fixes #456 
Fixes #457 
Fixes #465

### CODE REVIEW

File: cime_config/testdefs/testmods_dirs/cam/outfrq1d_ltr/user_nl_cam [hunk 1]
```diff
@@ -4,5 +4,10 @@ nhtfrq=-24,-24,-24,-24,-24,-24,-24,-24,-24
 avgflag_pertape = 'A',  'I',  'I',  'A',  'A', 'I', 'A'
 inithist='ENDOFRUN'
 ionos_epotential_ltr=.true.
-ltr_files = '$DIN_LOC_ROOT/atm/waccm/amie_data/REMIX_3-4_Aug_2010.nc'
+ltr_files = '$DIN_LOC_ROOT/atm/waccm/amie_data/REMIX_3-4_Aug_2010_c210302.nc'
```
Comment (cacraigucar) [inline cime_config/testdefs/testmods_dirs/cam/outfrq1d_ltr/user_nl_cam:7]: This file needs to be added to the svn repository

**Review by cacraigucar — APPROVED** [review]

File: doc/ChangeLog [hunk 2]
```diff
@@ -1,5 +1,142 @@
 ===============================================================
 
+Tag name:
+Originator(s): fvitt
+Date:
+One-line Summary: Misc updates for WACCM-X
+Github PR URL:
```
Comment (gold2718) [inline doc/ChangeLog:7]:
```suggestion
Github PR URL: https://github.com/ESCOMP/CAM/pull/474
```

**Review by gold2718 — APPROVED** [review]
I have a question which might need a code change (logic removal) but otherwise, it looks good!

File: src/ionosphere/waccmx/adotv_mod.F90 [hunk 3]
```diff
@@ -82,12 +82,7 @@ subroutine calc_adotv(z, un, vn, wn, adotv1, adotv2, adota1, adota2, &
                    dvec(i,j,2,2) * vn(k,i,j) + &
                    dvec(i,j,3,2) * wn(k,i,j))
            end do ! k=lev0,lev1-1
-        end do
-     end do
 
-     do j = lat0, lat1
-        if (j==jspole .or. j==jnpole) cycle
```
[the remark this answers is not in the document]
Comment (fvitt) [reply src/ionosphere/waccmx/adotv_mod.F90:89]: The line was erroneous.  We should not be skipping over the geographic pole here.  This change fixes issue #456.

**Review by nusbaume — APPROVED** [review]

**Review by peverwhee — APPROVED** [review]

**Pull request merged**